### ISSUE TITLE
Footer Design

### ISSUE THREAD

**Issue by ferindesigns** [unstructuredstudio/zubhub#591]
The change of language drop-down menu's colour contrasts with the colour of the footer. Also, the smaller words' font weight should be reduced or the font weight for the bigger ones should be increased.
![Screenshot 2023-03-08 012056](https://user-images.githubusercontent.com/108564127/223586747-0d33dd22-92e6-4ade-bca7-0298e214bd56.png)

**Comment by ferindesigns** [comment]
For one solution, I suggest increasing the boldness of the bigger text and reducing the space a little between the logo and the sections.
<img width="1440" alt="Untitled" src="https://user-images.githubusercontent.com/108564127/223593294-a98c2c5e-40ac-40f3-b1f6-b46f0b286ea6.png">

**Comment by KramStyles** [comment]
Hello @ferindesigns If this issue is approved, I'd love to work on it.

**Comment by ferindesigns** [comment]
Hi @KramStyles I'm also an applicant waiting to work on it.

**Comment by ferindesigns** [comment]
Hi @tuxology what do you think about the footer design?

**Comment by tuxology** [comment]
@ferindesigns some ideas are worth considering. Since this is a low priority, we will revisit this later. Thanks for filing

**Comment by srish** [comment]
Seems no more applicable
